components/dashboard/src/hooks/use-analytics-tracking.ts

```diff
@@ -13,6 +13,11 @@ export const useAnalyticsTracking = () => {
 
     // listen and notify Segment of client-side path updates
     useEffect(() => {
+        //store current path to have access to previous when path changes
```

Comment:
Moved this into here instead of it being in the user-loader hook for some reason.